x/storage/keeper/keeper.go

```diff
@@ -117,14 +117,6 @@ func (k Keeper) CreateBucket(
 		return sdkmath.ZeroUint(), errors.Wrap(types.ErrNoSuchStorageProvider, "the storage provider is not in service")
 	}
 
-	// check primary sp approval
-	if opts.PrimarySpApproval.ExpiredHeight < uint64(ctx.BlockHeight()) {
```

Comment:
these code cant be deleted directly. Need to modify after new hardfork is introduced.

Reply:
Why can't be deleted directly. With the new Go-SDK, verification no longer goes through SP to obtain, thus there is no expiredHeight.
In the old Go-SDK, creating a bucket would go through SP to get approval, but this is no longer necessary for ExpiredHeight verification.

Reply:
fixed with IsUpgraded(upgradetypes.Pawnee)
the correct hard fork version will be updated before release